e2e/msa/msaKeyManagementDelete.test.ts

```diff
@@ -111,7 +111,8 @@ describe('MSA Key management: delete keys and retire', function () {
     });
   });
 
-  it('should allow retiring MSA after additional keys have been deleted and tokens withdran', async function () {
+  // This test hangs under some unknown circumstances in parallel mode
+  it.skip('should allow retiring MSA after additional keys have been deleted and tokens withdran', async function () {
```

Comment:
Forgot to remove the skip?

Reply:
In my big Schemas PR, I was going through a lot of the e2e tests that need to be updated due to the schema grant changes, and I found quite a few missing `await`s; could be that's causing things to hang randomly.

Reply:
Is there a lint we can turn on for missing await in an async function?